### PR TITLE
Feature/int 1135

### PR DESCRIPTION
**Pull request by CJ-Polarity**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -43,4 +43,7 @@ key/
 *.pfx
 *.key
 *.pem
-*.cer
\ No newline at end of file
+*.cer
+
+# Ignore package-lock.json
+package-lock.json
```
Comment (sarus) [inline .gitignore:49]: @CJ-Polarity We shouldn't ignore the package-lock.  We specifically want to check it into the repo so we can have reproducible builds.

File: config/config.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ module.exports = {
    * @optional
    */
   description: 'Polarity Sharepoint integration',
-  entityTypes: ['IP', 'hash', 'domain', 'string', 'email'],
+  entityTypes: ['IP', 'hash', 'domain', 'string', 'email', 'cve'],
```
Comment (sarus) [inline config/config.js:28]: Can you also expand the `hash` type:

```
'MD5', 'SHA1', 'SHA256'
```

**Review by sarus — CHANGES_REQUESTED** [review]

File: config/config.js (hunk 2 above)
Comment (CJ-Polarity) [reply config/config.js:28]: Expand or replace?

Comment (sarus) [reply config/config.js:28]: @CJ-Polarity replace.

Basically we want to stop using the short hand forms for entity types that cover multiple types.

So no more usage of `hash`, IP`, and `*`.

**Review by sarus — APPROVED** [review]

**Pull request merged**